### PR TITLE
init

### PR DESCRIPTION
**Pull request by fenwer**

### CODE REVIEW

File: src/main/java/rest/Application.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    public static void main(String args[]) {
+        SpringApplication.run(Application.class);
+    }
+
+    @Override
+    public void run(String... strings) throws Exception {
+
+        //TODO: disable if no problem with cert
+        SSLUtil.turnOffSslChecking();
```
Comment (alovak) [inline src/main/java/rest/Application.java]: Can we add ca certificate so we don't need to turn off ssl check. Here you can get certificate https://github.com/payfort/start-php/blob/master/src/data/ca-certificates.crt

**Comment by fenwer** [comment]
added the ability

**Comment by fenwer** [comment]
you should have environment variables JAVA_HOME and JRE_HOME

**Comment by fenwer** [comment]
like 
export JAVA_HOME=/opt/jdk/
export JRE_HOME=/opt/jdk/jre/

**Comment by fenwer** [comment]
new jar in attach message on the upwork

**Comment by fenwer** [comment]
new jar on the upwork

**Pull request merged**